yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/create/SessionSpecTest.java

```diff
@@ -271,18 +273,27 @@ public void buildWithSubject() throws IOException {
                 mapper.writeValueAsString(session.getSubject()).getBytes(DEFAULT_CHARSET)
         );
 
-        assertThat(json.get("subject_id").asText(), is(Matchers.equalTo(subject.get(Property.SUBJECT_ID))));
+        assertThat(json.get("subject_id").asText(), is(Matchers.equalTo(subject.get(TestProperty.SUBJECT_ID))));
     }
 
-    private static final class Property {
+    @Test
+    public void shouldBuildWithResourceContainer() {
+        SessionSpec sessionSpec = SessionSpec.builder()
+                .withResources(resourceCreationContainerMock)
+                .build();
+
+        assertThat(sessionSpec.getResources(), is(resourceCreationContainerMock));
+    }
+
+    private static final class TestProperty {
```

Comment:
I don't understand the renaming
I feel like it would make more sense to split them in 2 classes
```
    private static final class IdentityProperty {

        private static final String TYPE = "type";
        private static final String SCHEME = "scheme";
        private static final String OBJECTIVE = "objective";
        private static final String TRUST_FRAMEWORK = "trust_framework";

        private IdentityProperty() { }

    }

    private static final class SubjectProperty {

        private static final String SUBJECT_ID = "subject_id";

        private SubjectProperty() { }

    }
```

Or something that make sense. `TestProperty` sounds like constants specific to the tests

Reply:
The rename came from an accidental refactor, where Intellij complained that the class didn't match a specific regex.  Agree that we should probably split them out, I'll do that now :+1: